airflow/jobs/base_job.py

```diff
@@ -192,7 +192,7 @@ def heartbeat(self):
                 self.heartbeat_callback(session=session)
                 self.log.debug('[heartbeat]')
         except OperationalError as e:
-            self.log.error("Scheduler heartbeat got an exception: %s", str(e))
+            self.log.exception("%s heartbeat got an exception: %s", self.__class__, str(e))
```

Comment:
This will also include the stack trace of the exception - was that intended?

Reply:
Ya, already printing it previously and just a small touch up to use `log.exception`( also removed the redundant `str(e)` in the draft).